### ISSUE TITLE
Releasing version 4 of PerfectHTTP and PerfectHTTP-Server?

### ISSUE THREAD

**Issue by uros-gardasevic** [PerfectlySoft/Perfect#302]
Hi, we recently moved from precompiled Perfect to one resolved via SPM.

We stumbled upon the crash explained in:

https://github.com/PerfectlySoft/Perfect/issues/296

only when running in Release configuration. On the very first request, app crashes in 

`return UnsafePointer(bytesArray).withMemoryRebound(to: Int8.self, capacity: bytesArray.count) {
	return String(validatingUTF8: $0)
}`

Now this seems to be fixed in PerfectLib from version 4.0 onwards by following commit:

https://github.com/PerfectlySoft/Perfect/commit/135fb33471f1dd316eb323f8a94d701bde2efa85#diff-69391fd63fbd9d34d8f320a64858e895ea4ab70dc9e87fafe0cfe701de2db5b0L250

The problem is that no publicly available version of HTTP and HTTP-Server depends on PerfectLib 4.0, which essentially is letting me with no access to the fix for this crash.

Long story short, is there an intent from your side to publish HTTP/HTTP-Server version 4.0 that depends on PerfectLib 4.0?

Thanks in advance.
Uros
